src/layout_abstract_syntax.rs

```diff
@@ -20,6 +20,11 @@ impl LayoutAbstractNode {
 pub struct LayoutAbstractTag {
 	pub namespace: String,
 	pub name: String,
+	/// Layout attributes, which are used by the layout engine.
+	pub layout_attributes: LayoutAttributes,
+	/// The special content attribute, representing the inner elements of this tag.
+	pub content: Option<AttributeValue>,
```

Comment:
I've added a `content` field where we can store the node content. I think it's better this way, instead of searching for it linearly.

Reply:
Now that we're breaking attributes into `layout_attributes`, `user_attributes`, (and `event_attributes` in the future for `on:` prefixes) I quite like this change!